opm/simulators/timestepping/SimulatorReport.cpp

```diff
@@ -73,70 +73,69 @@ namespace Opm
 
     void SimulatorReportSingle::reportStep(std::ostringstream& ss) const
     {
-        ss << "Time step summary: ";
         if (total_well_iterations != 0) {
-            ss << "well its = " << std::setw(2) << total_well_iterations << ", ";
+            ss << "Well its=" << std::setw(2) << total_well_iterations << ", ";
         }
-        ss << "newton its = " << std::setw(2) << total_newton_iterations << ", "
-           << "linearizations = "  << std::setw(2) << total_linearizations
-           << " ("  << std::fixed << std::setprecision(3) << std::setw(6) << assemble_time << " sec), "
-           << "linear its = " << std::setw(3) << total_linear_iterations
-           << " ("  << std::fixed << std::setprecision(3) << std::setw(6) << linear_solve_time << " sec)";
+       ss << "Newton its=" << std::setw(2) << total_newton_iterations << ", "
```

Comment:
If you are interested the `fmtlib`alternative would be something like:
```
auto msg = fmt::format("Newton its={:2}, lineraizations={:2} ({:2.1f}sec), linear its={:3} (:2.1f)sec)", 
                        total_linearizations, 
                        assemble_time, 
                        total_linear_iterations, 
                        linear_solve_time);
```

Reply:
Thanks for the tip, I can put that in.